shared/src/androidMain/kotlin/com/innova/innovaPoolingLibrary/DateFormatter.kt

```diff
@@ -0,0 +1,21 @@
+package com.innova.innovaPoolingLibrary
+
+import android.util.Log
+import java.text.SimpleDateFormat
+import java.util.*
+
+actual class DateFormatter {
+    var selectedDate = Date()
+    actual fun getFormattedDate(): String {
+        var formattedDate = ""
+        try {
+            formattedDate =
+                SimpleDateFormat(dateUtilities.outputDateFormat, Locale.getDefault()).format(
+                    selectedDate
+                )
+        } catch (e: Exception) {
+            Log.e("Simple date format exception occur due to -> ", e.toString())
```

Comment:
I think we should throw the exception here instead of just logging it. @harshasuda what are your thoughts on this?

Reply:
yes, log should be removed instead of returning the String you can return String?